addons/popochiu/engine/objects/gui/components/dialog_text/dialog_caption/dialog_caption.gd

```diff
@@ -4,14 +4,40 @@ extends PopochiuDialogText
 #region Private ####################################################################################
 func _modify_size(msg: String, _target_position: Vector2) -> void:
 	var _size := await _calculate_size(msg)
-	
+
 	# Define size and position (before calculating overflow)
+	rich_text_label.size.x = _size.x
 	rich_text_label.size.y = _size.y
+	rich_text_label.position.x = (get_viewport_rect().size.x/2) - (_size.x /2)
```

Comment:
I think the ideal approach in this case is not to modify the width (`size.x`) and X position of the **RichTextLabel**. Considering how this **Control** is configured in Godot, what we should do is ensure that the value defined in the `wrap_width` property is assigned to the `size.x` property of the **RichTextLabel** in the `_ready()` function. To control the Y position of the component, Godot's anchors can be used.